extensions/wikia/SitemapXml/SitemapXmlController.class.php

```diff
@@ -213,22 +216,22 @@ private function isRequestedGzipped() {
 		return strpos( $acceptEncoding, 'gzip' ) !== false;
 	}
 
-	private function parseSubpage( $subpage ) {
+	private function parsePath( $path ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)